var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -14,14 +14,15 @@ class Mvapich2(AutotoolsPackage):
     platforms (x86 (Intel and AMD), ARM and OpenPOWER)"""
 
     homepage = "https://mvapich.cse.ohio-state.edu/userguide/userguide_spack/"
-    url = "https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-2.3.6.tar.gz"
+    url = "https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-2.3.7.tar.gz"
     list_url = "https://mvapich.cse.ohio-state.edu/downloads/"
 
     maintainers = ['natshineman', 'harisubramoni', 'ndcontini']
 
     executables = ['^mpiname$']
 
     # Prefer the latest stable release
+    version('2.3.7', sha256='c39a4492f4be50df6100785748ba2894e23ce450a94128181d516da5757751ae')
```

Comment:
Can you resolve the conflict? I think the only change wrt `develop` is the new sha256.